packages/fx-core/src/common/tools.ts

```diff
@@ -487,13 +487,21 @@ export async function askSubscription(
 }
 
 // Determine whether feature flag is enabled based on environment variable setting
-export function isFeatureFlagEnabled(featureFlagName: string): boolean {
+export function isFeatureFlagEnabled(featureFlagName: string, defaultValue = false): boolean {
   const flag = process.env[featureFlagName];
-  // can enable feature flag by set environment variable value to "1" or "true"
-  if (flag && (flag === "1" || flag.toLowerCase() === "true")) {
-    return true;
+  if (flag === undefined) {
+    return defaultValue; // allows consumer to set a default value when environment variable not set
+  } else {
+    if (flag === "1" || flag.toLowerCase() === "true") {
+      return true; // can enable feature flag by set environment variable value to "1" or "true"
+    } else {
+      return false;
+    }
   }
-  return false;
+}
+
+export function isArmSupportEnabled(): boolean {
+  return isFeatureFlagEnabled("TEAMSFX_ARM_SUPPORT", false);
```

Comment:
false can be omitted

Reply:
yes, but here aims to turn off the feature flag explicitly. So I don't need to care what's the default value provided by `isFeatureFlagEnabled`. Imaging someone changed the default value of `isFeatureFlagEnabled` to true, but I always want this method to return false by default.

Reply:
OK, I know